spec/import_spec.rb

```diff
@@ -112,22 +130,16 @@ def upload_file!(name, ext='csv')
         end
       end
 
-      context "Win1251" do
-
-        before do
-          upload_file!(:authors_win1251_win_endline)
-        end
-
-        it "should import file with many records" do
-          expect(page).to have_content "Successfully imported 2 authors"
-          expect(Author.count).to eq(2)
-          Author.all.each do |author|
-            expect(author).to be_valid
-            expect(author.name).to be_present
-            expect(author.last_name).to be_present
-          end
-        end
+      context "auto detect encoding" do
+        include_examples 'successful inserts',
+                         :auto,
+                         :authors_win1251_win_endline
+      end
 
+      context "Win1251" do
+        include_examples 'successful inserts',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.